_opensearch/snapshots/searchable_snapshot.md

```diff
@@ -0,0 +1,119 @@
+{: .warning }
+
+A searchable snapshot is an index where data is read from a [snapshot repository]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#register-repository) on-demand at search time, rather than downloading all index data to cluster storage at restore time. Because the index data remains in the snapshot format in the repository, searchable snapshot indexes are inherently read-only. Any attempt to write to a searchable snapshot index will result in an error.
+
+To enable the searchable snapshots feature, reference the steps below.
+
+## Enabling the feature flag
+
+There are several methods for enabling searchable snapshots, depending on the install type.
+
+### Enable on a node using a tarball install
+
+The flag is toggled using a new jvm parameter that is set either in `OPENSEARCH_JAVA_OPTS` or in config/jvm.options.
+
+- Option 1: Update config/jvm.options by adding the following line:
+
+    ```json
+    -Dopensearch.experimental.feature.searchable_snapshot.enabled=true
+    ```
+
+- Option 2: Use the `OPENSEARCH_JAVA_OPTS` environment variable:
+
+    ```json
+    export OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.searchable_snapshot.enabled=true"
+    ```
+- Option 3: For developers using Gradle, update run.gradle by adding the following lines:
+
+    ```json
+    testClusters {
+      runTask {
+        testDistribution = 'archive'
+        if (numZones > 1) numberOfZones = numZones
+        if (numNodes > 1) numberOfNodes = numNodes
+        systemProperty 'opensearch.experimental.feature.searchable_snapshot.enabled', 'true'
+      }
+    }
+    ```
+
+- Finally, create a node in your opensearch.yml file and define the node role as `search`:
+
+    ```bash
+    node.name: snapshots-node
+    node.roles: [ search ]
+    ```
+
+### Enable with Docker containers
+
+If you're running Docker, add the following line to docker-compose.yml underneath the `opensearch-node` and `environment` section:
+
+```json
+OPENSEARCH_JAVA_OPTS="-Dopensearch.experimental.feature.searchable_snapshot.enabled=true" # Enables searchable snapshot
+```
+
+To create a node with the `search` node roll, add the line `- node.roles: [ search ]` to your docker-compose.yml file:
+
+```bash
+version: '3'
+services:
+  opensearch-node1:
+    image: opensearchproject/opensearch:2.4.0
+    container_name: opensearch-node1
+    environment:
+      - cluster.name=opensearch-cluster
+      - node.name=opensearch-node1
+      - node.roles: [ search ]
+```
+
+## Create a searchable snapshot index
+
+Creating a searchable snapshot index is done by specifying the `remote_snapshot` storage type using the [restore snapshots API]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore/#restore-snapshots).
+
+Request field | Description
```

Comment:
Should "field" be capitalized? I believe we capitalize other multi-word table headings.